### PR TITLE
finding number of digits using logs created with unit tests

### PR DESCRIPTION
**Pull request by vicky1999**
num_digits.py was created to find the number of digits in a number using log10 method in math module.
This PR was created from #659 

Implementation and unit tests were done.

### CODE REVIEW

File: algorithms/maths/num_digits.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"""
+   num_digits() method will return the number of digits of a number in O(1) time using math.log10() method.
+"""
+
+import math
+
+def num_digits(n):
+  if(n==0):
```
Comment (ankit167) [inline algorithms/maths/num_digits.py:8]: Please handle the case if n < 0

File: tests/test_maths.py [hunk 2]
```diff
@@ -448,5 +450,18 @@ def test_find_order_simple(self):
...
+    """[summary]
+    Test for the file num_digits.py
+
+    Arguments:
+        unittest {[type]} -- [description]
+    """
+    def test_num_digits(self):
+        self.assertEqual(2,num_digits(12))
+        self.assertEqual(5,num_digits(99999))
+        self.assertEqual(1,num_digits(8))
```
Comment (ankit167) [inline tests/test_maths.py:463]: Add test case for:
1. n = 0
2. n < 0

**Review by ankit167 — CHANGES_REQUESTED** [review]

File: tests/test_maths.py (hunk 2 above)
Comment (vicky1999) [reply tests/test_maths.py:463]: Sure.  I'll do that

**Comment by vicky1999** [comment]
the test case for n<0 has been handled and the unit tests were created for:
- n=0
- n<0

**Review by ankit167 — APPROVED** [review]

**Pull request merged**